client/src/components/Landing.style.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  background-color: var(--highlight);
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+
+export const H1 = styled.h1`
+  position: absolute;
```

Comment:
I think we might have to have a look at another way to make this work instead of ```position absolute```. This doesn't seem to work too well on different screens, because mine looks like this right now:
![Screen Shot 2019-06-21 at 7 12 08 PM](https://user-images.githubusercontent.com/9094166/59942771-815e8900-9458-11e9-8ff6-4e02a4672834.png)
And this on mobile:
![Screen Shot 2019-06-21 at 7 12 15 PM](https://user-images.githubusercontent.com/9094166/59942782-88859700-9458-11e9-8302-1b1f5e6b6f86.png)
I do think you're close though, because your main element has a flex display! I think we can use that to put everything in the right place without having to specify too much. Happy to help you look into this at some point!